### PR TITLE
More float nodes

### PR DESCRIPTION
**Pull request by jdudetv**
Tweaked the zoom to be smoother it got a bit jumpy at small scales and added in some util nodes to make stuff work / things that wern't fully implemented.

### CODE REVIEW

File: packages/src/obs/requests.ts [hunk 1]
```diff
@@ -1253,11 +1253,18 @@ export function register(pkg: Package<EventTypes>, { obs }: Ctx) {
...
+          inputName: ctx.getInput(io.inputName),
+          inputVolumeDb: ctx.getInput(io.inputVolumeDb),
+        });
+      }
+      if (ctx.getInput(io.inputVolumeDb) === 0) {
+        obs.call("SetInputVolume", {
+          inputName: ctx.getInput(io.inputName),
+          inputVolumeMul: ctx.getInput(io.inputVolumeMul),
+        });
+      }
```
Comment (Brendonovich) [inline packages/src/obs/requests.ts]: These should be separate nodes

File: packages/src/fs.ts [hunk 2]
```diff
@@ -25,6 +25,8 @@ export function register(actions: { list(path: string): Promise<Entry[]> }) {
     async run({ ctx, io }) {
       const files = await actions.list(ctx.getInput(io.path));
 
+      console.log(files);
```
Comment (Brendonovich) [inline packages/src/fs.ts]: remove

**Review by Brendonovich — CHANGES_REQUESTED** [review]

File: packages/src/obs/requests.ts (hunk 1 above)
Comment (jdudetv) [reply packages/src/obs/requests.ts]: made into two nodes

File: packages/src/fs.ts (hunk 2 above)
Comment (jdudetv) [reply packages/src/fs.ts]: removed

**Review by Brendonovich — APPROVED** [review]

**Pull request merged**